internal/generate/testdata/go/deploy/olm-catalog/memcached-operator/manifests/memcached-operator.clusterserviceversion.yaml

```diff
@@ -90,7 +90,7 @@ spec:
                       fieldPath: metadata.name
                 - name: OPERATOR_NAME
                   value: memcached-operator
-                image: quay.io/example/memcached-operator:v0.0.2
+                image: quay.io/example/memcached-operator:v0.0.3
```

Comment:
Any reason this is getting bumped?

Reply:
Because `/testdata/go/deploy/operator.yaml` has image set to `quay.io/example/memcached-operator:v0.0.3`. So just changed this, instead of that.